main/acle.md

```diff
@@ -2362,7 +2362,7 @@ following:
 
 [^fmv-note-ignore]: The intention is to support the usecase of newer code if
   compiled with an older compiler. The compiler may provide diagnostic message
-  and could block the compilation for example in case of `-fpendantic`.
+  and could block the compilation for example in case of `-fpedantic`.
```

Comment:
In fact the compiler option is called `-pedantic`. Without the `f`.

The commit message and the pull request title also need correcting.